src/data/registry.json

```diff
@@ -6,11 +6,14 @@
       "did": "did:sov:2NPnMDv5Lh57gVZ3p3SYu3",
       "logo_url": "https://s3.eu-central-1.amazonaws.com/builds.eth.company/absa.svg",
       "domain": "www.absa.africa",
-      "registered_at": "2023-05-24T18:14:24",
-      "updated_at": "2023-05-24T18:14:24",
```

Comment:
Why remove these properties?

Reply:
As you see they're defined dynamically now.